edb/server/compiler/dbstate.py

```diff
@@ -156,6 +156,7 @@ class QueryUnit:
     # Set only when a query is compiled with "json_parameters=True"
     in_type_args: typing.Optional[typing.Tuple[str, ...]] = None
 
+    system_config: bool = False
```

Comment:
Please add a comment explaining that this is only set to True when this QueryUnit contains a `CONFIGURE SYSTEM` command.